source/install.txt

```diff
@@ -70,108 +46,52 @@ Installation
        name: Windows
        content: |
 
-         .. note::
-
-            The Windows installation of |compass| requires
-            **Microsoft .NET Framework version 4.5 or later.** The
-            |compass-short| installer prompts you to install the
-            minimum required version of the .NET framework if it is
-            not already installed on your system.
+         1. Double-click the installer file to install |compass-short|.
+            By default, the installer extracts |compass-short| to the
+            following path:
```

Comment:
AFAIK you can't modify the install path. This wording makes it sound like you can. I might consider not using `By default` here. cc @gilberto-mdb